scripts/prepare_sector_network.py

```diff
@@ -703,6 +644,68 @@ def define_spatial(nodes, options):
 
     spatial.co2.df = pd.DataFrame(vars(spatial.co2), index=nodes)
 
+    # oil
+
+    spatial.oil = SimpleNamespace()
+
+    if options["oil"]["spatial_oil"]:
+        spatial.oil.nodes = nodes + " oil"
+        spatial.oil.locations = nodes
+    else:
+        spatial.oil.nodes = ["Africa oil"]
+        spatial.oil.locations = ["Africa"]

```

Comment:
We can take the change to replace this into World/Earth oil instead of Africa.
The same applies below